src/erliam_srv.erl

```diff
@@ -80,55 +75,109 @@ terminate(_Reason, _State) ->
 code_change(_OldVsn, State, _Extra) ->
     {ok, State}.
 
-
 %%%% INTERNAL FUNCTIONS
 
 maybe_update_credentials() ->
     case ets:lookup(?TAB, credentials) of
-        [Credentials] ->
-            MinLifetime = ?MIN_LIFETIME,
-            case remaining_lifetime(Credentials) of
-                N when N =< MinLifetime ->
-                    update_credentials();
-                _ ->
-                    ok
-            end;
-        [] ->
-            update_credentials()
+      [Credentials] ->
+          MinLifetime = ?MIN_LIFETIME,
+          case remaining_lifetime(Credentials) of
+            N when N =< MinLifetime ->
```

Comment:
case indentation is 2 and the statement body is actually 4, is that expected?

Reply:
Well, it seems that that's the case for all cases, so, I guess it's expected

Reply:
It's expected and configurable: it's called `sub_indent`, the default is `2` but we can use `4` to match the rest of the indentation.

Reply:
We should be consistent across projects, tho… and we're using the default for spillway, mero, etc…